rust/auction/program/src/processor.rs

```diff
@@ -606,6 +706,17 @@ impl BidState {
             BidState::OpenEdition { bids, max } => None,
         }
     }
+
+    pub fn has_instant_bid(&self, instant_sale_amount: u64) -> bool {
```

Comment:
i was confused by this naming - minor nit - rename it to "lowest_winning_bid_is_instant_bid_price"